scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -160,8 +161,26 @@ def run_robot_test(self, command, handler):
 
 class Console(Harness):
 
+    def get_testcase_name(self):
+        '''
+        Get current TestCase name.
+        Console Harness id has only TestSuite id without TestCase name suffix.
+        TODO: Only the first TestCase name is taken if available.
+        '''
+        if self.instance is not None and len(self.instance.testcases) == 1:
```

Comment:
in python shall we just use below?
```suggestion
        if self.instance and self.instance.testcases:
```

Reply:
this workaround is applicable with a single Ztest testcase only.
I've simplified this `if` statement; also better description is added for this method as well as for the commit message.